admin/grants.sql

```diff
@@ -680,4 +680,46 @@ GRANT USAGE, MONITOR
     TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE.SYNAPSE_RAW_ALL_DEVELOPER;
 GRANT USAGE, MONITOR
     ON SCHEMA SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_RAW
-    TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_RAW_ALL_DEVELOPER;
\ No newline at end of file
+    TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_RAW_ALL_DEVELOPER;
+
+-- Grant schema privileges for SYNAPSE_AGGREGATE schema
+-- SYNAPSE_DATA_WAREHOUSE
+GRANT OWNERSHIP
+	ON SCHEMA SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE
+	TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE_ALL_ADMIN;
+GRANT USAGE, MONITOR
+    ON SCHEMA SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE
+    TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE_ALL_ANALYST;
+GRANT USAGE, MONITOR
+    ON SCHEMA SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE
+    TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE_ALL_DEVELOPER;
+GRANT OWNERSHIP
+	ON ALL TABLES
+	IN SCHEMA SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE
+	TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE_ALL_ADMIN;
+-- SYNAPSE_DATA_WAREHOUSE_DEV
+GRANT OWNERSHIP
+    ON SCHEMA SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_AGGREGATE
+    TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_AGGREGATE_ALL_ADMIN;
+GRANT USAGE, MONITOR
+    ON SCHEMA SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_AGGREGATE
+    TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_AGGREGATE_ALL_ANALYST;
+GRANT USAGE, MONITOR
+    ON SCHEMA SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_AGGREGATE
+    TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_AGGREGATE_ALL_DEVELOPER;
+GRANT OWNERSHIP
+	ON ALL TABLES
+	IN SCHEMA SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_AGGREGATE
+	TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_AGGREGATE_ALL_ADMIN;
+
+-- Grant table read privileges in SYNAPSE_AGGREGATE schema
+-- SYNAPSE_DATA_WAREHOUSE
+GRANT SELECT, REFERENCES
+	ON ALL TABLES
```

Comment:
Same here: intuitively we wouldn't expect grants to current tables for a new schema, only future tables.

Reply:
Since `admin/grants.sql` is run every time we deploy to prod, it's sort of ambiguous what kind of comments to include here. If I include some information relevant to the _first_ time we run this line, the same comment will become meaningless for the next 100+ deployments 🤷 

Anyway -- I have no quarrel with adding this comment.